docs/pages/index.mdx

```diff
@@ -61,11 +61,13 @@ export default async function Home(props) {
 
   return (
     <FrameContainer postUrl="/frames" state={state} previousFrame={previousFrame}>
-      <FrameImage src="https://picsum.photos/seed/frames.js/1146/600" />
-      <FrameButton onClick={dispatch}>
-        {state.count}
-      </FrameButton>
-    </FrameContainer>
+      <FrameImage>
+        <div tw="w-full h-full bg-black text-white justify-center items-center">
```

Comment:
whats tw? can we not make it className?

Reply:
it's how satori uses tailwind

Reply:
hm it doesn't support normal classNames stuff?

Reply:
might confuse people without a comment

Reply:
can we make default bg `bg-slate-700` or something to make it less harsh